src/polytopes/ngon.jl

```diff
@@ -78,7 +83,8 @@ signarea(::Triangle{3}) = error("signed area only defined for triangles embedded
 
 function normal(t::Triangle{3})
   A, B, C = t.vertices
-  ((B - A) × (C - A)) / 2
+  n = (B - A) × (C - A) / 2
+  Vec(ustrip.(n) * unit(lentype(t)))
```

Comment:
we need to review this later after sorting out the the role of `Vec` and `coordinates`